membership-attribute-service/test/acceptance/AccountControllerAcceptanceTest.scala

```diff
@@ -149,7 +164,7 @@ class AccountControllerAcceptanceTest extends AcceptanceTest {
       val giftSubscriptionFromSubscriptionService = TestSubscription(
         id = Subscription.Id(giftSubscription.Id),
         name = Subscription.Name(giftSubscription.Name),
-        plans = CovariantNonEmptyList(TestPaidSubscriptionPlan(product = Product.SupporterPlus), Nil),
+        plans = List(TestPaidSubscriptionPlan(productRatePlanId = TestCatalog.digipackPrpId)),
```

Comment:
this test was a bit confusing, it was testing
- a gift supporter plus, a contribution, and a patron.
Since you can only have a gift digi sub (or GW) I changed it to test a more realistic situation
- gift digisub, a supporter plus and a patron.